css/style.css

```
@@ -0,0 +1,288 @@
+
+
+}
+
+/* Global */
+
+.page-body {
+  min-width: 1200px;
+  margin: 0;
+  padding: 0;
+  font-family: "PT Sans", Arial, sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: 700;
+  color: var(--basic-light-black);
+  text-transform: uppercase;
+  background-color: var(--basic-white);
+  background-image: url("/img/background_index.jpg");
+  background-position: top center;
+  background-repeat: no-repeat;
+}
+
+a {
+  text-decoration: none;
+}
+
+img {
+  max-width: 100%;
+  height: auto;
+}
+
+/* Main navigation */
+
+.main-navigation {
+  background-color: var(--basic-white);
+}
+
+/* Site navigation */
+
+.site-navigation {
+  line-height: 26px;
+  font-weight: bold;
+  color: var(--basic-black);
+  list-style: none;
+}
+.site-navigation a {
+  color: var(--basic-black);
+}
+
+.site-navigation a:hover {
+  color: var(--basic-blue);
+}
+
+.site-navigation a:active {
+  color: var(--basic-black);
+  mix-blend-mode: normal;
+  opacity: 0.3;
+}
+
+.site-navigation .current{
+  color: var(--special)
+}
+
+/* Features */
+
+.features {
+  color: var(--basic-black);
+  list-style: none;
+  text-align: center;
+  line-height: 26px;
+}
+.features p {
+  font-style: normal;
+  font-weight: normal;
+
+}
+
+.features h1{
+  font-size: 21px;
+  font-style: normal;
+  font-weight: bold;
+
+
+}
+
+.features-list {
+  list-style: none;
+}
+
+
+.feature-item h3, {
+  font-size: 21px;
+  line-height: 21px;
+  font-weight: bold;
+}
+
+
+.feature-item {
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: normal;
+  color: var(--basic-light-black);
+  background-color: var(--basic-white);
+  list-style: none;
+  text-align: center;
+}
+ .box {
+  color: var(--basic-white);
+  background-color: var(--basic-blue);
+ }
+
+
+/* Button */
+
+.button,
+.button-catalog {
+  font: inherit;
+  text-align: center;
+  font-size: 14px;
+  line-height: 21px;
+  color: var(--basic-white);
+  vertical-align: middle;
+  text-transform: uppercase;
+  background-color: var(--basic-blue);
+  border: none;
+}
+.button:hover {
+background-color: var(--basic-blue);
+}
+
+.button:focus {
+  background-color: var(--basic-blue-focus);
+}
+
+.button:active {
+  background-color: var(--basic-blue-active);
+}
+
+.button-catalog {
+  background-color: var(--special);
+}
+
+.button-catalog:hover {
+  background-color: var(--special);
+}
+
+.button-catalog:focus {
+  background-color: var(--special-focus);
+}
+
+.button-catalog:active {
+  background-color: var(--special-active);
+}
+/* Contacts */
+
+.contacts-phone {
+  color: inherit;
+}
+
+.contacts {
+  font-style: normal;
+  font-weight: bold;
+  font-size: 21px;
+  line-height: 26px;
+}
+/* Main footer */
+
+.main-footer {
+  color: var(--basic-black);
+  background-color: rgba(255, 255, 255, 0.9);
+  align-items: center;
+  text-align: center;
+
+}
+
+.footer-contacts a {
+  color: var(--basic-black);
+  text-decoration: underline;
+}
+
+.footer-social li{
+  list-style: none;
```

обнулять стили лучше сразу у самого списка (ul/ol):
```suggestion
  list-style: none;
  margin: 0;
  padding: 0;
```